README.md

```diff
@@ -1,89 +1,114 @@
-# Cloudbeat
 [![Coverage Status](https://coveralls.io/repos/github/elastic/cloudbeat/badge.svg?branch=main)](https://coveralls.io/github/elastic/cloudbeat?branch=main)
 [![Go Report Card](https://goreportcard.com/badge/github.com/elastic/cloudbeat)](https://goreportcard.com/report/github.com/elastic/cloudbeat)
-[![Build Status](https://internal-ci.elastic.co/buildStatus/icon?job=cloudbeat%2Fcloudbeat-mbp%2Fmain)](https://internal-ci.elastic.co/job/cloudbeat/job/cloudbeat-mbp/job/main/)
 
-### Cloudbeat evaluates cloud assets for security compliance and ships findings to Elasticsearch
+# Cloudbeat
 
-## Table of contents
-- [Prerequisites](#prerequisites)
-- [Running Cloudbeat](#running-cloudbeat)
-  - [Clean up](#clean-up)
-  - [Remote Debugging](#remote-debugging)
-  - [Skaffold Workflows](#skaffold-workflows)
-- [Code guidelines](#code-guidelines)
+Cloudbeat analyzes cloud assets for security compliance and sends findings to Elasticsearch as part of
+the [Cloud Security Posture](https://www.elastic.co/blog/secure-your-cloud-with-elastic-security) plugin in Kibana.
+
+## Getting Started
+
+To get started with Cloud Security Posture on your cluster, see
+our [documentation](https://www.elastic.co/guide/en/security/master/get-started-with-kspm.html#kspm-setup-unmanaged).
+
+- [Setup KSPM for Amazon EKS clusters](https://www.elastic.co/guide/en/security/master/get-started-with-kspm.html#kspm-setup-unmanaged)
+
+- [Setup KSPM for unmanaged Kubernetes clusters](https://www.elastic.co/guide/en/security/master/get-started-with-kspm.html#kspm-setup-eks-start)
+
+___
+
+## Cloudbeat Development
+
+### Table of contents
 
+- [Prerequisites](#prerequisites)
+- [Deploying Cloudbeat as a process](#deploying-cloudbeat)
+    - [Unmanaged Kubernetes](#clean-up)
+    - [Amazon Elastic Kubernetes Service (EKS)](#amazon-elastic-kubernetes-service-(EKS))
+- [Deploying Cloudbeat with Elastic-Agent](#running-cloudbeat-with-elastic-agent)
 
 ## Prerequisites
```

Comment:
`Cloudbeat Development` (`h2`) is the section title, so `Prerequisites` needs to be an `h3`, not `h2`